sdk/python/feast/client.py

```diff
@@ -448,7 +448,12 @@ def list_feature_sets(
         if name is None:
             name = "*"
 
-        filter = ListFeatureSetsRequest.Filter(project=project, feature_set_name=name)
+        if labels is None:
```

Comment:
Would it make more sense to set the default to `dict()`? You are setting the default to `None` and then setting it to `{}`

Reply:
Yup that makes sense, will change to default {}

Reply:
Updated.